workflow/tests/base_results/results_simulations_hvac.csv

```diff
@@ -421,7 +421,7 @@ base-simcontrol-timestep-10-mins-occupancy-stochastic-10-mins.xml,6.8,91.76,3600
 base-simcontrol-timestep-10-mins-occupancy-stochastic-60-mins.xml,6.8,91.76,36000.0,24000.0,0.0,31783.0,8654.0,7508.0,0.0,575.0,6571.0,0.0,0.0,1738.0,2171.0,4566.0,0.0,0.0,19895.0,6102.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,615.0,0.0,3320.0,0.0,0.0,147.0,0.0,-653.0,0.0,800.0
 base-simcontrol-timestep-10-mins.xml,6.8,91.76,36000.0,24000.0,0.0,31783.0,8654.0,7508.0,0.0,575.0,6571.0,0.0,0.0,1738.0,2171.0,4566.0,0.0,0.0,19895.0,6102.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,615.0,0.0,3320.0,0.0,0.0,147.0,0.0,-653.0,0.0,800.0
 base-simcontrol-timestep-30-mins.xml,6.8,91.76,36000.0,24000.0,0.0,31783.0,8654.0,7508.0,0.0,575.0,6571.0,0.0,0.0,1738.0,2171.0,4566.0,0.0,0.0,19895.0,6102.0,7037.0,0.0,207.0,321.0,0.0,0.0,0.0,2293.0,615.0,0.0,3320.0,0.0,0.0,147.0,0.0,-653.0,0.0,800.0
-base-zones-spaces-multiple.xml,6.8,91.76,36000.0,24000.0,0.0,30312.0,8922.0,5506.0,0.0,575.0,6835.0,0.0,0.0,1738.0,2171.0,4566.0,0.0,0.0,16662.0,3478.0,5579.0,0.0,207.0,571.0,0.0,0.0,0.0,2293.0,615.0,0.0,3920.0,0.0,0.0,447.0,0.0,-653.0,0.0,1100.0
+base-zones-spaces-multiple.xml,6.8,91.76,36000.0,24000.0,0.0,30328.0,8938.0,5506.0,0.0,575.0,6835.0,0.0,0.0,1738.0,2171.0,4566.0,0.0,0.0,16594.0,3409.0,5579.0,0.0,207.0,571.0,0.0,0.0,0.0,2293.0,615.0,0.0,3920.0,0.0,0.0,447.0,0.0,-653.0,0.0,1100.0
```

Comment:
Small bugfix for room-by-room calculations where the above-grade portion of the walls wasn't being incorporated in the WAR.